scripts/ci/create-docker-image.sh

```diff
@@ -25,7 +26,7 @@ function cross_compile() {
     docker run -tid --name skydive-crosscompile -v `pwd`:/root/go/src/github.com/skydive-project/skydive -v $GOPATH/.cache/govendor:/root/go/.cache/govendor ubuntu:18.04 /bin/bash
     trap cleanup ERR
     docker exec --env http_proxy=http://172.17.0.1:3128 skydive-crosscompile /root/go/src/github.com/skydive-project/skydive/scripts/ci/create-docker-multiarch-image.sh $1 $2 $3
-    docker build -t ${DOCKER_IMAGE}:${arch}-${DOCKER_TAG} -f contrib/docker/Dockerfile.${arch} contrib/docker/
+    docker build -t ${DOCKER_IMAGE}:${arch}-${DOCKER_TAG} --build-arg TARGET_ARCH="${arch}" -f contrib/docker/Dockerfile.ubuntu contrib/docker/
```

Comment:
I would prefer to keep using CentOS as the base image

Reply:
For s390x, we could use s390x/ubuntu though, or s390x/clefos (use CentOS as base)

Reply:
> I would prefer to keep using CentOS as the base image for x86_64

sure - haven't changed this - native x86_64 builds still use Dockerfile

> or s390x/clefos (use CentOS as base)

thanks - didn't know about clefos

I've been playing today with a cross-build docker file (obviously based off create-docker-multiarch-image.sh) and a scratch based image (is any of the OS used? I've really only just got this built).

https://github.com/grooverdan/skydive/commit/e23c7502f17c6bb0800d70c0c3383f8b77708472

pushed to dockerhub danielblack/skydive:latest

I had trouble installing a clang/llc compiler into the docker image to make WITH_EBPF=true possible. It seemed to conflict the cross complier packages. Have I missed something obvious that has enabled create-docker-multiarch-image.sh to build this target?
<pre>
$ docker run -ti skydive-crosscompile-ppc64le bash 
root@69a59c78b5f1:~/go/src/github.com/skydive-project/skydive# apt install clang-6.0 llvm-6.0
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 clang-6.0 : Depends: libstdc++-7-dev but it is not going to be installed
             Depends: libc6-dev but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
</pre>

Reply:
> sure - haven't changed this - native x86_64 builds still use Dockerfile
Ok, my bad.

> I've been playing today with a cross-build docker file (obviously based off create-docker-multiarch-image.sh) and a scratch based image (is any of the OS used? I've really only just got this built).
We used to have scratch based image but due to name resolution issues - even using netgo -, we moved to distroless

Regarding clang, did you try to use "apt install clang:ppc64el llvm:ppc64el" ?

Reply:
> we moved to distroless

Ok. I did do a FR to move distroless to multiarch.

> did you try to use "apt install clang:ppc64el llvm:ppc64el"

I did, and it also had conflicts. It wouldn't help as its a x86_64 -> ppc64le cross compile that's required rather than a native ppc64le compile. 

It seems from the existing docker build logs that the flow.o is from the x86_64 compile (ok as its -march=bpf) and reused in the subsequent cross-compiles.